$NetBSD: patch-source4_auth_gensec_wscript__build,v 1.1 2022/08/30 17:47:50 jperkin Exp $

Do not mark gensec_krb5 as internal, kdc.so ends up linked against it and fails
shlibs check as gensec_module_krb5.so has not been installed.

--- source4/auth/gensec/wscript_build.orig	2022-07-27 07:18:22.200870300 +0000
+++ source4/auth/gensec/wscript_build
@@ -14,7 +14,6 @@ bld.SAMBA_MODULE('gensec_krb5',
 	subsystem='gensec',
 	init_function='gensec_krb5_init',
 	deps='samba-credentials authkrb5 com_err',
-	internal_module=False,
         enabled=bld.AD_DC_BUILD_IS_ENABLED()
 	)
 
